roles/provision-mdc-server-apb/tasks/provision-service-server.yml

```diff
@@ -40,7 +40,7 @@
       - --provider=openshift
       - --openshift-service-account={{ mobile_developer_console_sa_name }}
       - --upstream=http://localhost:{{ mobile_developer_console_port }}/
-      - --openshift-sar=[{"namespace":"{{ namespace }}","resource":"secrets","name":"mdc-server","verb":"list"}, {"namespace":"{{ namespace }}","resource":"buildconfig","name":"mdc-server","verb":"create"}]
+      - --openshift-sar=[{"namespace":"{{ namespace }}","resource":"secrets","name":"mdc-server","verb":"list"}]
```

Comment:
@wei-lee  This caused an issue with logging in via oauth proxy for a `developer` user (default user created with `oc cluster up`).

The problem was caused by the resource name `buildconfig` - it is not recognised by oauth proxy (resources should be always used in plural; `buildconfigs` works in this case).
This was not a problem for user that has cluster-admin privileges - you can type anything into a resource name and it will work. It is just ignored during login process, - as I mentioned - only for a user with cluster-admin rights.

But since we decided to hide build tab by default, I think it does not make sense to check user's access to buildconfig within the project now. Wdyt?

Reply:
@psturc yes, agreed. I have some ideas to refactor how the login is done: rather than check the permission on login, we can get the user token and forward in the requests to the openshift API server. This way the permission checks on the login should be minimum.